wrapper/src/main/java/software/amazon/jdbc/util/monitoring/Monitor.java

```diff
@@ -17,9 +17,11 @@
 package software.amazon.jdbc.util.monitoring;
 
 public interface Monitor {
-  void start();
+  void execute();
```

Comment:
I'd personally prefer `start()` and `stop()`. It's cleaner for me and it shows that monitor runs for a long time. `execute()` sounds like a quick run and get result.